comictaggerlib/volumeselectionwindow.py

```diff
@@ -474,6 +474,16 @@ def categorize(result: ComicVolume) -> int:
         self.twList.setSortingEnabled(True)
         self.twList.selectRow(0)
         self.twList.resizeColumnsToContents()
+        # Get the width of the issues, year and publisher columns
+        owidth = self.twList.columnWidth(1) + self.twList.columnWidth(2) + self.twList.columnWidth(3)
```

Comment:
There is a margin on the right which is lost on the publisher column. Adding about 15 seems to be about right:
```
owidth = self.twList.columnWidth(1) + self.twList.columnWidth(2) + (self.twList.columnWidth(3) + 15)
```

Reply:
Fixed by using:
```python
self.twList.contentsRect().width() - owidth
```